scheduling_resources/scheduling-logging-resources.adoc

```diff
@@ -0,0 +1,44 @@
+include::_attributes/common-attributes.adoc[]
```

Comment:
This needs to follow our style **[guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata)**

Reply:
done